src/c/admin-balance-transfer-item.js

```diff
@@ -0,0 +1,58 @@
+import m from 'mithril';
+import h from '../h';
+
+const adminBalanceTransferItem = {
+    view(ctrl, args) {
+        let item = args.item;
+        return m('.w-row', [
+            m('.w-col.w-col-1.w-col-tiny-1', [
+                m('.w-checkbox.w-clearfix', [
+                    m('input.w-checkbox-input[type=\'checkbox\']', {
+                        disabled: (item.state != 'pending'),
+                        checked: args.listWrapper.isSelected(item.id),
+                        onchange: (event) => {
+                            if(event.currentTarget.checked) {
+                                args.listWrapper.selectItem(item);
+                            } else {
+                                args.listWrapper.unSelectItem(item);
+                            }
+                        }
+                    })
+                ]),
+            ]),
+            m('.w-col.w-col-3', [
+                m('.fontsize-smaller.fontweight-semibold.lineheight-tighter', [
+                    `${item.user_name}`,
+                    m('span.fontcolor-secondary.fontsize-smallest',
+                      `(${item.user_public_name})`),
+                ]),
+                m('.fontcolor-secondary.fontsize-smallest',
+                  item.user_email),
+                m('.fontcolor-secondary.fontsize-smallest',
+                  `USER_ID: ${item.user_id}`)
+             ]),
+            m('.w-col.w-col-2', [
+                m('span.fontsize-small', `R$ ${h.formatNumber(item.amount, 2, 3)}`)
+            ]),
+            m('.w-col.w-col-2.w-hidden-small.w-hidden-tiny', [
+                m('', [
```

Comment:
empty m(''). we should either just remove this wrapper and return the array or make it an actual tag.